addons/website/migrations/13.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,27 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_model_renames = [
+    ('website.redirect', 'website.rewrite'),
+]
+
+_table_renames = [
+    ('website_redirect', 'website_rewrite'),
+]
+
+_field_renames = [
+    ('website.rewrite', 'website_rewrite', 'type', 'redirect_type'),
+]
+
+_xmlid_renames = [
+    ('website.access_website_redirect', 'website.access_website_rewrite'),
+    ('website.access_website_redirect_designer', 'website.access_website_rewrite_designer'),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_models(env.cr, _model_renames)
+    openupgrade.rename_tables(env.cr, _table_renames)
+    openupgrade.rename_fields(env.cr, _field_renames)

```

Comment:
Get this error on migration

```
Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/service/server.py", line 1179, in preload_registries
    registry = Registry.new(dbname, update_module=update_module)
  File "/opt/odoo/custom/src/odoo/odoo/modules/registry.py", line 86, in new
    odoo.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/opt/odoo/custom/src/odoo/odoo/modules/loading.py", line 479, in load_modules
    force, status, report, loaded_modules, update_module, models_to_check, upg_registry)
  File "/opt/odoo/custom/src/odoo/odoo/modules/loading.py", line 363, in load_marked_modules
    upg_registry=upg_registry,
  File "/opt/odoo/custom/src/odoo/odoo/modules/loading.py", line 190, in load_module_graph
    migrations.migrate_module(package, 'pre')
  File "/opt/odoo/custom/src/odoo/odoo/modules/migration.py", line 185, in migrate_module
    migrate(self.cr, installed_version)
  File "/usr/local/lib/python3.7/site-packages/openupgradelib/openupgrade.py", line 1778, in wrapped_function
    if use_env2 else cr, version)
  File "/opt/odoo/auto/addons/website/migrations/13.0.1.0/pre-migration.py", line 26, in migrate
    openupgrade.rename_fields(env.cr, _field_renames)
  File "/usr/local/lib/python3.7/site-packages/openupgradelib/openupgrade.py", line 595, in rename_fields
    cr = env.cr
  File "/opt/odoo/custom/src/odoo/odoo/sql_db.py", line 163, in wrapper
    return f(self, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/sql_db.py", line 443, in __getattr__
    return getattr(self._obj, name)
AttributeError: 'psycopg2.extensions.cursor' object has no attribute 'cr'
```
I think rename_fields just expects env, not env.cr